server/src/main/java/com/broadcom/lsp/cobol/core/model/VarFinder.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package com.broadcom.lsp.cobol.core.model;
+
+import com.broadcom.lsp.cobol.core.model.variables.Variable;
+import com.google.common.collect.Lists;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.groupingBy;
+
+/**
+ * The class take all defined variables and search thought them by partial qualifier
+ */
+public class VarFinder {
```

Comment:
Please, avoid using "-er" endings for classes when it is not a pattern name or a subject area entity

Reply:
Eh, I liked that name so much.

Reply:
Come on, now it sounds like a special variable type